### PR TITLE
HAAR-1193 Set up External users web clients

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/auth/AuthApiService.kt [hunk 1]
```diff
@@ -11,15 +11,15 @@ import java.util.UUID
 
 @Service
 class AuthApiService(
-  @Qualifier("authWebClientUtils") val authWebClientUtils: WebClientUtils
+  @Qualifier("authWebClientUtils") val serviceWebClientUtils: WebClientUtils
 ) {
```
Comment (karenmillermoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/auth/AuthApiService.kt:15]: just renamed to make easier to understand

File: src/main/resources/application-dev.yml [hunk 2]
```diff
@@ -34,21 +34,21 @@ nomis:
...
     client-secret: clientsecret
 
 external-users:
   endpoint:
     url: http://localhost:8088
     timeout: 1s
   client:
-    client-id: manage-users-ext-users-client
+    client-id: manage-users-api-ext-users-client
     client-secret: clientsecret
```
Comment (karenmillermoj) [inline src/main/resources/application-dev.yml:46]: renamed all these to match auth and set up in dev

File: src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/config/AuthWebClientConfiguration.kt [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.digital.hmpps.manageusersapi.config
+
```
Comment (karenmillermoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/config/AuthWebClientConfiguration.kt:2]: This file is just the old WebClientConfig file (not sure why it doesn't show as rename)

**Review by AlTurner-MOJ — APPROVED** [review]
lgtm

**Pull request merged**